### DISCUSSION TITLE
tinc can't be configured in router mode

### DISCUSSION THREAD

**Discussion by flosoft** [Freetz-NG/freetz-ng#795] [Allgemein]
It seems like it's not possible to change the following 2 values of `tinc.conf`.

```
DeviceType    = tun
Mode          = router
```

Every time I reboot the router, the file `/tmp/flash/tinc/tinc.conf` gets reset - even after using `modsave`.

I've tried modifying the file `/mod/etc/default.tinc/tinc_conf` but it's not possible to rewrite it.

Any thoughts on what the best thing to do is?

**Comment by flosoft** [comment]
The workaround for now that I've done is to update this before building the firmware. What would be the way to add this as an option to the UI?

**Comment by fda77** [comment]
The values are static currently: https://github.com/Freetz-NG/freetz-ng/blob/master/make/pkgs/tinc/files/root/etc/default.tinc/tinc_conf

>  What would be the way to add this as an option to the UI?

Like the other variables in tinc_conf, eg `$TINC_SERVER0`
